Rakefile

```diff
@@ -17,3 +17,23 @@ task :console do
   ARGV.clear
   Pry.start
 end
+
+task :add_exchange do
+  exchange_name = ARGV.last
```

Comment:
may want to snakecase this?

Reply:
ya, if want PascalCase I will edit later

Reply:
can do

```
next if exchange_name.nil?
```